test/pubtator/pubtator.js

```diff
@@ -216,4 +218,43 @@ describe('pubtator', function () {
       }); // BioC Document
     }); // forEach
   }); // map
+  
+  describe('get', function(){
+    it('Should be null when no body returned', async function(){
+      let pmid = '38496625';
+      var path = new RegExp( pmid );
+      nock( NCBI_BASE_URL )
+        .get( path )
+        .reply( 200 );
+      let bioCDocument = await pubtator.get( pmid );
+      expect( bioCDocument ).to.be.null;
+    });
+
+    it('Should exist when a body is returned', async function(){
+      let pmid = '28041912';
+      var path = new RegExp( pmid );
+      nock( NCBI_BASE_URL )
+        .get( path )
+        .reply( 200, { foo: 'bar' } );
+      let bioCDocument = await pubtator.get( pmid );
+      expect( bioCDocument ).to.exist;
+    });
+  }); // get
+
+  describe('hints', function(){
+    it('Should be null when invalid publicationXref provided', async function(){
+      let bioCDocument = await pubtator.hints( { dbPrefix: 'bar', id: 'foo'} );
+      expect( bioCDocument ).to.be.null;
+    });
+
+    it('Should exist when a body is returned', async function(){
+      let pmid = '28041912';
+      var path = new RegExp( pmid );
+      nock( NCBI_BASE_URL )
+        .get( path )
+        .reply( 200, { foo: 'bar' } );
+      let bioCDocument = await pubtator.get( pmid );
```

Comment:
Isn't this testing `get`?

Reply:
Yes it tests `get` ,  I took it as it is from branch `iss1148_pubtator-service`